Sources/RingPublishingTracking/Private/Networking/Models/Artemis.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  ArtemisID.swift
+//  RingPublishingTracking
+//
+//  Created by Adam Mordavsky on 09.11.23.
+//
+
+import Foundation
+
+public struct ArtemisObject: Codable {
```

Comment:
should not be public (as it is in private directory), better to have separate public model

Reply:
and for public stuff, each property and object should have a comment saying that it is